tests/expose_test.go

```diff
@@ -70,6 +74,18 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		return job
 	}
 
+	cleanupService := func(serviceName string, namespace string) func() error {
+		return func() error {
```

Comment:
We need to block on the removal, otherwise we can fail to create another service with the same name while this one is being deleted. Although removal of a Service should be swift.

Reply:
Done.